src/backend/_scss/application.scss

```diff
@@ -659,7 +659,79 @@ a {
 
 .track-names {
   color: $black;
-  margin-top: 16%;
+  margin-top: 0%;
+}
+
+.social-buttons {
+  float: right;
+  margin-bottom: 100px;
+  margin-right: 10px;
+  margin-top: 15px;
+}
+
+.social-button {
+  border-radius: 5px;
+  color: $white;
+  cursor: pointer;
+  display: inline-block;
+  height: 40px;
+  margin-right: 3px;
+  text-align: center;
+  width: 40px;
+
+  i {
+    height: 100%;
+    line-height: 40px;
+    width: 100%;
+  }
+}
+
+.facebook {
+  background-color: $facebook;
+}
+
+.twitter {
+  background-color: $twitter;
+}
+
+.google {
+  background-color: $google;
+}
+
+@media (max-width: 990px) {
+
+  .title-legend {
+    float: left;
+    width: 40%;
+  }
+
+  .track-names {
+    margin-top: 0;
+  }
+
+  .social-buttons {
```

Comment:
@geekyd Not getting this. According to markup  `social-button class` should be nested  inside `social-buttons`